tutor/src/screens/teacher-dashboard/index.jsx

```diff
@@ -89,6 +89,11 @@ class TeacherDashboardWrapper extends React.Component {
     if (course.currentRole.isTeacherStudent && course.roles.teacher) {
       course.current_role_id = null;
     }
+
+    // getting the reference book data to show chapters and sections in the plan details
+    if (!course.referenceBook.ensureLoaded()) {
+      course.referenceBook.fetch();
+    }
   }
```

Comment:
same here, you can just put a call to `course.referenceBook.ensureLoaded()`,no need for the if